src/Take.Blip.Builder/Actions/ProcessHttp/ProcessHttpAction.cs

```diff
@@ -103,9 +105,9 @@ private void AddUserToHeaders(HttpRequestMessage httpRequestMessage, IContext co
         /// <param name="context"></param>
         private void AddBotIdentityToHeaders(HttpRequestMessage httpRequestMessage, IContext context)
         {
-            if (ShouldAddHeader("processHttpAddBotIdentityToRequestHeader", context))
+            if (ShouldAddHeader(ADD_BOT_KEY, context))
```

Comment:
This is not about your PR really, but this method name is kind of misleading: it seems that ADD_BOT_KEY is the header to be added...
What about change to something like `ConfigurationFlagIsEnabled`? This could even be an extension over `Flow` class...

Reply:
Moved to `Flow`